src/App.jsx

```diff
@@ -1,33 +1,54 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
+function getVisibleMovies(movies, query) {
+  if (query) {
+    return movies.filter(movie => (
+      movie.title.toLowerCase().includes(query.toLowerCase().trim())
+      || movie.description.toLowerCase().includes(query.toLowerCase().trim())
+    ));
+  }
 
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+  return movies;
+}
+
+export const App = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = getVisibleMovies(moviesFromServer, query);
+  const newText = (event) => {
```

Comment:
```suggestion
  const handleChange = (event) => {
```
follow to convention of function naming. first word should be a verb which tell what fuction is doing